src/index.js

```diff
@@ -1,8 +1,8 @@
 // Global CSS variables
-import './shared/colors.css';
-import './shared/sizing.css';
-import './shared/variables.css';
-import './shared/reset.css';
+// import './shared/colors.css';
```

Comment:
I'm almost certain this will break other imports. We should build a test case on a bare react app that imports all our components, and run using `yarn link` locally to verify.

Reply:
I say this ^ because I moved the css import to here from a css module file after discovering that buttons (and I think another component as well) weren't taking any styles when imported into another project.

Reply:
Ah, you're right. I'd forgotten about the buttons. I'll just remove the reset for now since that's the main culprit

Reply:
After my change, you'll need to fix small conflicts around the reset.css. 

Otherwise: 🌻🥪